runtime/astar/src/lib.rs

```diff
@@ -890,10 +890,6 @@ pub type Executive = frame_executive::Executive<
     frame_system::ChainContext<Runtime>,
     Runtime,
     AllPalletsWithSystem,
-    (
-        pallet_assets::migration::v1::MigrateToV1<Runtime>,
-        pallet_balances::migration::MigrateToTrackInactive<Runtime, xcm_config::CheckingAccount>,
-    ),
```

Comment:
I don't think this is a good idea - these migrations haven't been executed on Astar yet.

Reply:
Yes, but we didn't plan apply this runtime before applying this migrations, keep migrations here hold our release process until migrations applied to astar.

Reply:
The migration code is designed in such a way that it won't be executed unless it's applicable.

So theoretically we could keep these migrations "forever" without causing any issues.

Reply:
I see, I’ll not remove it next time.